doc/protocol.md

```diff
@@ -153,63 +151,63 @@ Exclusion criteria:
 
 ### Stopping criteria
 
-In terms of database searches, we run the final search 01/10/22. Hand searching
+In terms of database searches, we aim to run the final search by 01/10/22. Searches
 will be concluded when we reach conceptual saturation (i.e., when it is
 determined that we are not identifying any new concepts/resources). The first
-1000 records will be screened on Google.
+1000 records will be screened on Google. In the interests of feasibility, a maximum for 4 books will be selected and evaluated. Books will be selected based on a consensus in the research group.
 
 DANIEL: why not search all the references? Do we want to have two people
 review each paper or is it okay we search a database each? For this
 study, I would think that it is okay not to have two reviewers, as it is
 not a systematic reivew. It is more important to have more databases,
 then double check each reference.
+Hannah: I think I added this "1000 records" idea based on my discussions with the research librarian, who had done something similar. I'm happy to search all references. I agree with your proposal to search a database each, although maybe we should trial this together to ensure that we are using a similar approach/decision-making method.
 
 ## Analysis Plan
 
 ### Data analysis approach
 
+Data from all searches in online databases will be exported into text files or
+CVS files for further processing including removal of duplicates. Google Sheets
+will be used to manage records including screening decisions.
+
 One author will extract data using a standardized and tested template via Google
 Sheets. Data regarding the data source (e.g., author, title, publciation year),
 open collaboration practices, and any other relevant information, will be
 extracted. A second author will check extracted data. Extracted data will be
 summarized with the descriptive analytical method described by Arksey and
-O'Malley (2005).
+O'Malley (2005). The descriptive
+analytical method is aimed at identifying and summarizing different open
+collaboration practices.
 
 TODO: Decide on how we will extract data 
 Daniel: we could also use a
 Google sheet instead - that is easier to share. If we use covidence, we
 can also use the data extraction form there.
+Hannah: Happy to use Google Sheets. I think Covidence would only be appropriate if we were doing a systematic review. I do have access to Covidence, though, if we decide to go down that route.
```

Comment:
As much as possible we should use openly accessible software and services. So that would exclude using Covidence.